content/concepts/kata/translations.md

```diff
@@ -64,9 +65,9 @@ Before pending translation is accepted and new language version becomes availabl
 
 After review is completed, reviewer can finalize it with one of following actions:
```

Comment:
- "After review is completed" --> "After a review is completed"
- "reviewer can ..." --> "the reviewer can ..."

Reply:
```suggestion
After a review is completed, the reviewer can finalize it with one of following actions:
```